english/references/references.md

```diff
@@ -1,3 +1,5 @@
+Chief Marshmellow sees you put down Obama as a reference. So her calls him and Obama says [you used to call me on my cellphone](https://youtu.be/hrjs1UXC8rU)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.